src/reporting/OperationsBase.ts

```diff
@@ -0,0 +1,97 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import isomorphicFetch from 'cross-fetch';
+import { DataType, ECProperty } from './interfaces/mappingInterfaces/GroupProperties';
+
+const ACCEPT = "application/vnd.bentley.itwin-platform.v1+json";
+
+export class OperationsBase {
+
+  public readonly fetch = isomorphicFetch;
+  public readonly basePath;
+
+  constructor(basePath?: string) {
+    this.basePath = basePath ?? "https://api.bentley.com/insights/reporting";
+  }
+
+  /**
+   * Creates a request body and headers
+   * @param {string} operation string specifying which opperation will be performed
+   * @param {string} accessToken OAuth access token with scope `insights:read`
+   * @param {string} content request body
+   * @memberof OperationsBase
+   */
+  public createRequest(operation: string, accessToken: string, content?: string): RequestInit {
+    const request: RequestInit = {
+      method: operation,
+      headers: {
+        Authorization: String(accessToken),
```

Comment:
Do not use `String` constructor anywhere. `accessToken` is already a string.

Reply:
removed all string constructors